spec/asyncapi.md

```diff
@@ -1326,12 +1383,13 @@ Example using Avro to define the payload:
   "payload": {
     "schemaFormat": "application/vnd.apache.avro+json;version=1.9.0",
     "schema": {
-      "$ref": "path/to/user-create.avsc#/UserCreate"
+      "$ref": "./user-create.avsc"
```

Comment:
Please make consistent the usage of quotes vs double quotes.

Reply:
Can you please elaborate on the locations? As far as I've seen, the strings in YAML are single quoted, and JSON examples are double quoted.

Reply:
I've made these changes which I found inconsistent: c788faf

Reply:
Addressed more issues I could find via b60d353